### PR TITLE
Turn on test coverage checking.

### PR DESCRIPTION
**Pull request by jacobsi**

### CODE REVIEW

File: project/BalboaService.scala [hunk 1]
```diff
@@ -29,12 +30,13 @@ object BalboaService {
 }
 
 /**
- * Java Messaging Service currently using ActiveMQ... Soon to be deprecated and removed.
```
Comment (TranquilMarmot) [inline project/BalboaService.scala]: 😆

**Comment by TranquilMarmot** [comment]
👍  :shipit:

**Comment by jacobsi** [comment]
retest this please

**Comment by jacobsi** [comment]
retest this please

**Pull request merged**